training/03_Infrastructure_Deployment/02_What_is_in_a_config.adoc

```diff
@@ -1,10 +1,14 @@
 
 = What is in a Config?
 
-This section models the typical layout of a config using a-base-config as an example.
+First let's define what a config is. 
+
+In our developments, a config is the skel of any kind of technical asset (a lab, a demo, a workshop) to be deployed on the cloud of our choice (remember AgnosticD is based on the Infrastructure as Code paradigm for deploying architectures).
```

Comment:
Hmmm, this does not read well to me. A config is a lot more than a skeleton of anything. A config is everything. It is the thing that is deployed.

Also the next sentence...I would not describe AgnosticD as such and you don't deploy architectures, really. 

I think this rewording needs to go back to the drawing board. Or others can chime in.

Reply:
Happy to have that reworded, I do appreciate those sentences to be reworded by you, as I'm not a native speaker nor an expert developer, @stencell, if you don't mind